src/components/App/App.jsx

```diff
@@ -10,16 +12,17 @@ import NotFound from '../NotFound/NotFound';
 const App = () => {
   const location = useLocation();
   const isAdoptPage = location.pathname === '/adoptable-creatures';
+  const isAboutPage = location.pathname === '/about';
 
   return (
     <main className='App'>
-      <Header isAdoptPage={isAdoptPage}>
-        {isAdoptPage ? null : (
-          <AdoptableCreaturesNavButton />
-        )}
+      <Header isAdoptPage={isAdoptPage} isAboutPage={isAboutPage}>
+        {isAdoptPage ? null : (<AdoptableCreaturesNavButton />)}
```

Comment:
these look really clean, i love the use of everything in one line, it makes a big difference in the readability of code